README.md

```diff
@@ -34,6 +34,10 @@ Authors Haven app can be run by:
     Run the app according to the environment you need:
 
    #### Development environment:
+    `python manage.py makemigrations --sttings=authors.settings.dev`
```

Comment:
Check your spelling of settings on this line.

Reply:
Thanks, I have fixed it.